src/config.rs

```diff
@@ -7,18 +7,27 @@ use std::path::{Path, PathBuf};
 
 #[derive(Serialize, Deserialize)]
 pub struct Config {
+    #[serde(default, skip_serializing_if = "Option::is_none")]
```

Comment:
I personally think it's best that those are serialized and written to the config file because the people will be able to see all the options that exist.

I feel strongly against this change.